homeassistant/components/zwave/__init__.py

```diff
@@ -916,3 +918,18 @@ def refresh_from_network(self):
             return
         for value_id in dependent_ids + [self._value.value_id]:
             self._value.node.refresh_value(value_id)
+
+    @callback
+    def _schedule_update(self):
+        """Schedule delayed update."""
+        if not self._scheduled_update:
```

Comment:
Should be the inverse. We should not do anything if an update is already scheduled.